[VPCEP] fail creating service with no approval

With the latest devel, if using `approval_enabled` set to `false` for the service creation:
```
=== RUN   TestService_basic
=== PAUSE TestService_basic
=== CONT  TestService_basic
    resource_opentelekomcloud_vpcep_service_test.go:26: Step 1/2 error: After applying this test step, the plan was not empty.
        stdout:
        
        
        Terraform used the selected providers to generate the following execution
        plan. Resource actions are indicated with the following symbols:
          ~ update in-place
        
        Terraform will perform the following actions:
        
          # opentelekomcloud_vpcep_service_v1.service will be updated in-place
          ~ resource "opentelekomcloud_vpcep_service_v1" "service" {
              ~ approval_enabled = true -> false
                id               = "1a8939df-6ffc-4e10-a66b-befe791e3282"
                name             = "tf-test-4eGJ"
                tags             = {
                    "key" = "value"
                }
                # (5 unchanged attributes hidden)
        
                # (1 unchanged block hidden)
            }
        
        Plan: 0 to add, 1 to change, 0 to destroy.
--- FAIL: TestService_basic (45.10s)

FAIL

Process finished with the exit code 1
```

--- a/opentelekomcloud/acceptance/vpcep/resource_opentelekomcloud_vpcep_service_test.go
+++ b/opentelekomcloud/acceptance/vpcep/resource_opentelekomcloud_vpcep_service_test.go
@@ -143,6 +143,8 @@ resource "opentelekomcloud_vpcep_service_v1" "service" {
   vpc_id      = data.opentelekomcloud_vpc_subnet_v1.shared_subnet.vpc_id
   server_type = "LB"
 
+  approval_enabled = false
+
   port {
     client_port = 80
     server_port = 8080
@@ -169,7 +171,7 @@ resource "opentelekomcloud_vpcep_service_v1" "service" {
   vpc_id      = data.opentelekomcloud_vpc_subnet_v1.shared_subnet.vpc_id
   server_type = "LB"
 
-  approval_enabled = false
+  approval_enabled = true
 
   port {
     client_port = 80

--- a/opentelekomcloud/services/vpcep/resource_opentelekomcloud_vpcep_service_v1.go
+++ b/opentelekomcloud/services/vpcep/resource_opentelekomcloud_vpcep_service_v1.go
@@ -66,7 +66,7 @@ func ResourceVPCEPServiceV1() *schema.Resource {
 			"approval_enabled": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
+				Default:  true,
 			},
 			"service_type": {
 				Type:     schema.TypeString,
@@ -145,21 +145,19 @@ func resourceVPCEPServiceCreate(ctx context.Context, d *schema.ResourceData, met
 		return fmterr.Errorf(ErrClientCreate, err)
 	}
 
+	approvalEnabled := d.Get("approval_enabled").(bool)
 	opts := &services.CreateOpts{
-		PortID:      d.Get("port_id").(string),
-		PoolID:      d.Get("pool_id").(string),
-		VIPPortID:   d.Get("vip_port_id").(string),
-		ServiceName: d.Get("name").(string),
-		RouterID:    d.Get("vpc_id").(string),
-		ServiceType: services.ServiceType(d.Get("service_type").(string)),
-		ServerType:  services.ServerType(d.Get("server_type").(string)),
-		Ports:       getPorts(d),
-		TCPProxy:    d.Get("tcp_proxy").(string),
-		Tags:        common.ExpandResourceTags(d.Get("tags").(map[string]interface{})),
-	}
-	if v, ok := d.GetOk("approval_enabled"); ok {
-		enabled := v.(bool)
-		opts.ApprovalEnabled = &enabled
+		PortID:          d.Get("port_id").(string),
+		PoolID:          d.Get("pool_id").(string),
+		VIPPortID:       d.Get("vip_port_id").(string),
+		ServiceName:     d.Get("name").(string),
+		RouterID:        d.Get("vpc_id").(string),
+		ApprovalEnabled: &approvalEnabled,
+		ServiceType:     services.ServiceType(d.Get("service_type").(string)),
+		ServerType:      services.ServerType(d.Get("server_type").(string)),
+		Ports:           getPorts(d),
+		TCPProxy:        d.Get("tcp_proxy").(string),
+		Tags:            common.ExpandResourceTags(d.Get("tags").(map[string]interface{})),
 	}
 
 	svc, err := services.Create(client, opts).Extract()

